### PR TITLE
Issue 283

### PR DESCRIPTION
**Pull request by hamsterready**
With this in place one can use:

```
  def setArrayUUID(ps: PreparedStatement, x: List[UUID]) =
    ps.setArray(pos, new ClickHouseArray(Types.BINARY, x.map(toBytes).map(new String(_)).toArray)))
```

And since the type of the elements is BINARY it will NOT escape the serialized values (strings).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -100,6 +100,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```
Comment (serebrserg) [inline pom.xml]: I haven't found any usages of the dependency. Is it a mistake or have I missed something?

**Comment by hamsterready** [comment]
Ping?

File: pom.xml (hunk 1 above)
Comment (hamsterready) [reply pom.xml]: Hi @serebrserg I am pretty sure that I provided the explanation here and was waiting for you to come back. Looks like I pressed the wrong button.

I added this new dependency to support JDK9+ builds.

Without the dependency one will get:

```
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] To suppress warnings about obsolete options, use -Xlint:-options.
/home/hamster/Projects/clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/util/ClickHouseBlockChecksumTest.java:[7,21] error: package javax.xml.bind does not exist
[ERROR] /home/hamster/Projects/clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/util/ClickHouseBlockChecksumTest.java:[18,32] error: cannot find symbol
[ERROR]   symbol:   variable DatatypeConverter
  location: class ClickHouseBlockChecksumTest
/home/hamster/Projects/clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/util/ClickHouseBlockChecksumTest.java:[37,32] error: cannot find symbol
[INFO] 3 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.451 s
[INFO] Finished at: 2019-01-11T17:09:02+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:testCompile (default-testCompile) on project clickhouse-jdbc: Compilation failure: Compilation failure: 
[ERROR] To suppress warnings about obsolete options, use -Xlint:-options.
[ERROR] /home/hamster/Projects/clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/util/ClickHouseBlockChecksumTest.java:[7,21] error: package javax.xml.bind does not exist
[ERROR] /home/hamster/Projects/clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/util/ClickHouseBlockChecksumTest.java:[18,32] error: cannot find symbol
[ERROR]   symbol:   variable DatatypeConverter
[ERROR]   location: class ClickHouseBlockChecksumTest
[ERROR] /home/hamster/Projects/clickhouse-jdbc/src/test/java/ru/yandex/clickhouse/util/ClickHouseBlockChecksumTest.java:[37,32] error: cannot find symbol
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

```
And as you can see `ClickHouseBlockChecksumTest.java` has explicit dependency on `javax.xml.bind`.

**Comment by serebrserg** [comment]
Thanks. That is clear now. Moved to #294 to resolve conflicts and merge

**Pull request closed without merging**